lib/index.js

```diff
@@ -11,12 +12,23 @@ function drawCanvas(){
 }
 
 let character = new Character();
+let game = new Game(character);
 
 function drawSprite(){
   context.fillStyle = "green";
   context.fillRect(character.x, character.y, character.width, character.height);
 }
 
+function drawPortal(){
```

Comment:
Possible portal sprites to review at: http://spritefx.blogspot.com/2013/04/sprite-portals.html